ipatests/test_integration/test_replica_promotion.py

```diff
@@ -570,3 +570,24 @@ def test_replica_install_with_existing_entry(self):
         master.run_command(arg)
 
         tasks.install_replica(master, replica)
+
+
+class TestRecomendedReplicationAgreement(IntegrationTest):
+    """Maximum recomended number of agreements per replica is 4.
+    This test checks if number of agreement exceeds the recomendation,
+    warnning should be given to the user
+
+    related ticket : https://pagure.io/freeipa/issue/6533"""
+
+    num_replicas = 5
+
+    def test_recomended_replication_agreement(self):
+        tasks.install_topo('star', self.master, self.replicas,
+                           [], setup_replica_cas=False)
+        arg = ['ipa', 'topologysuffix-verify', 'domain']
+        cmd = self.master.run_command(arg)
+        expected_str1 = ("Recommended maximum number of " +
```

Comment:
No need to concat the strings.

Reply:
It is exceeding 80 characters if not concatenated.

Reply:
You can delete '+' and it will be as one string.

Reply:
What I mean is that:
```
("Recommended maximum number of "
"agreements per replica exceeded")
```
is still valid and considered one string, thus more lightweight.

Reply:
@mrizwan93 there is no need to use the `+` when a string exceeds the limits of chars per line. You can do this way:
```python
string = ("a very big big big  big  big  big  big  big  big  big  big  big big "
          "_ big string")
print(string)
>>> a very big big big  big  big  big  big  big  big  big  big  big big _ big string
```

Reply:
@Tiboris @stlaz @felipevolpone  thanks.